packages/pigeon/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.1.9
```

Comment:
Should this be 0.1.9+1? I think that's the convention for plugins for a patch version bump when < 0.x.x

Reply:
The +X is used for changes that change the repository, but don't change the binary in anyway, like updating a README.md.

Reply:
That's what I thought, but I don't think that's how the ecosystem team does it when it's < 1.0
https://github.com/flutter/plugins/pull/2090#discussion_r328265368